ftc/agents/CA.py

```diff
@@ -21,15 +21,15 @@ def get(self, fault_index):
 
 class CA():
     def __init__(self, B):
-        self.B = B
+        self.B = B.copy()
 
-    def get(self, fault_index):
+    def get(self, What, fault_index=()):
         """Notes
         `fault_index` should be 1d array, e.g., `fault_index = [1]`.
         """
-        self.B[:, fault_index] = np.zeros((4, 1))
-        BB = self.B
-        return BB
+        BB = self.B @ What
+        BB[:, fault_index] = np.zeros((4, 1))
+        return np.linalg.pinv(BB.dot(What))
```

Comment:
이거 맞나요..? 테스트로 시나리오들 돌려보고 있죠...?ㅠ

Reply:
넵! 확인 했는데요! 문제가 있나요?..

Reply:
```suggestion
        return np.linalg.pinv(BB)
```
이렇게 해야할 것 같아서... 아닌가..?

Reply:
죄송... 지금 0으로 모두 같은 값으로 나와서 그냥 넘어갈 뻔 했네요. 이게 맞아요!!